packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -67,11 +70,21 @@ abstract final class SemanticsService {
     await SystemChannels.accessibility.send(event.toMap());
   }
 
+  /// Checks if announce is supported on the given platform.
+  bool isAnnounceSupported();
```

Comment:
we should surface this through AccessibilityFeature
https://api.flutter.dev/flutter/dart-ui/AccessibilityFeatures-class.html

Reply:
in this pr or a different pr?

Reply:
cc @chunhtai 
I take Chun-Heng's feedback to mean in this PR, as in, we should not add this here, and instead incorporate it into AccessibilityFeature for consistent use across the framework. I'm grabbing some time for folks who know or are working in this area to sync up on this.